app/views/posts/show.html.erb

```diff
@@ -1 +1,14 @@
-<h1> Show me</h1>
\ No newline at end of file
+<h1> show post for a given user</h1>
+<div class="posts">
+  <div>
+    <p> <%= @post.text%> </p>
+    <p > Comments: <%= @post.comments_counter %>, Likes: <%= @post.likes_counter %> </p>
+  </div>
+  <div > 
+    <%= form_with url: "/users/#{@post.author_id}/posts/#{@post.id}/likes" do |f| %>
+  <%= f.submit "Like" %>
+  <% end %>
+  </div>
+</div>
+
+<%= link_to 'Add comment', new_user_post_comment_path(@user, @post) %>
```

Comment:
- Good job done so far, but the feature on this page does not display as it is in the wireframe. The `post_show` page should render all comments for that particular post. I am making this change a required one because this feature was implemented at your last milestone, `Views`, and also approved. Please fix this 🙏 
Wireframe:
![Fqff](https://github.com/Cabraham1/Blog/assets/73607512/52bcc1fd-2635-48f3-bc49-030b1058f4cf)

Your project:
![wrgwrgw](https://github.com/Cabraham1/Blog/assets/73607512/0c1c25c0-7b87-41b5-89a8-f33525845328)